### ISSUE TITLE
0.91.1 Upgrade - error setting up entry /dev/ttyUSB0 for zha

### ISSUE THREAD

**Issue by vinmar1** [zigpy/zigpy-deconz#30]
Just to recap environment

Conbee native ZHA on Intel NUC / Ubuntu 18.04.2 / Docker

0.91.0 was working great so I turned off debug log. After upgrading to 0.91.1 reboot my zigbee sseemed to stop working, all entities not found with this error.

Error setting up entry /dev/ttyUSB0 for zha
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/homeassistant/config_entries.py", line 303, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/usr/local/lib/python3.7/site-packages/homeassistant/components/zha/__init__.py", line 113, in async_setup_entry
    await zha_gateway.async_initialize(config_entry)
  File "/usr/local/lib/python3.7/site-packages/homeassistant/components/zha/core/gateway.py", line 91, in async_initialize
    await self.application_controller.startup(auto_form=True)
  File "/usr/local/lib/python3.7/site-packages/zigpy_deconz/zigbee/application.py", line 49, in startup
    await self._api.read_parameter(NETWORK_PARAMETER['nwk_panid'][0])
  File "/usr/local/lib/python3.7/site-packages/zigpy_deconz/api.py", line 195, in read_parameter
    timeout=COMMAND_TIMEOUT
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 423, in wait_for
    raise futures.TimeoutError()
concurrent.futures._base.TimeoutError

So I tried a power down restart including pulling the conbee and plugging it back in so I was sure it reset and a couple entities came back online with the power up but not all. Next I turned the debug logging back on and just did a restart from the config menu and this time everything was found and all appears to be working this time. I've attached the log anyway. I'll see what happens from here but maybe you can spot something from what I have collected.

[home-assistant.zip](https://github.com/zigpy/zigpy-deconz/files/3050423/home-assistant.zip)

**Comment by vinmar1** [comment]
Just a quick update... this could be incidental so if it looks like it is just close the ticket. The reason I say this is I added on the new Visual Studio and afterward did a restart from the config menu. I got the couple of warnings I always get on startup but everything looked fine. Here is one more clean startup file if you want to take a look at it.

[home-assistant 2.zip](https://github.com/zigpy/zigpy-deconz/files/3050453/home-assistant.2.zip)
